app/boards/shields/levinson/Kconfig.defconfig

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2021 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+ 
+ 
+if SHIELD_LEVINSON_LEFT
+
+config ZMK_KEYBOARD_NAME
+	default "Levinson Left"
+
+endif
+
+if SHIELD_LEVINSON_RIGHT
+
+config ZMK_KEYBOARD_NAME
+	default "Levinson Right"
+
+endif
```

Comment:
A recent change was made to how we define the central and peripheral for splits, which can be seen here: https://zmkfirmware.dev/docs/development/new-shield#kconfigdefconfig. Could you update this file to match the docs? Along with this you'll need to clear out the left and right `.conf` files.

Reply:
Changes made and pushed.